addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -139,6 +139,20 @@ function changePaddingSize(direction) {
     };
 }
 
+function clickOnDiscard(position = "bottom") {
+    return [{
+        trigger: "div:not(.o_loading_dummy) > #oe_snippets button[data-action=\"cancel\"]:not([disabled])",
+        extra_trigger: "body:not(:has(.o_dialog))",
+        content: Markup(_t("<b>Click Discard</b> to Discard all Changes.")),
+        position: position,
+    }, {
+        trigger: "iframe body:not(.editor_enable)",
+        noPrepend: true,
+        auto: true,
+        run: () => null,
+    }];
+}
```

Comment:
Don't they already have tours when they use the Discard button ? I'm surprised we need a new one.

Reply:
@dmo-odoo Yes! need to add a new one cause there is no tour yet about `Discard`